03-vectors/vectors-practice-2.Rmd

```diff
@@ -33,6 +33,10 @@ that showcases which ones have the best `mpg`, then, the ones that have the
 highest `hp`.  Using `order()`, rearrange this data frame such that it's ordered
 first by `mpg` (low to high) and then by `hp` (high to low).
 
+```{r, echo=FALSE}
+# Order by  ascending mpg and descending hp
+new.mtcars<-mtcars[order(mtcars$mpg, -mtcars$hp),]
```

Comment:
Using the - sign for high to low sorting is a nice feature I didn't know existed.